### PR TITLE
Fix `web-wagmi` build

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: examples/nextjs/components/auth/LoginButton.tsx [hunk 1]
```diff
@@ -1,39 +1,21 @@
 import { useWalletLogin, useWalletLogout } from '@lens-protocol/react-web';
 import { useEffect } from 'react';
 import toast from 'react-hot-toast';
-import { useAccount, useConnect, useDisconnect } from 'wagmi';
-import { InjectedConnector } from 'wagmi/connectors/injected';
```
Comment (desfero) [inline examples/nextjs/components/auth/LoginButton.tsx:5]: Simplifying `next.js` example given it was anyway broken as was using the older `wagmi 0.x`. We anyway have wagmi integration inside `web-wagmi` so to simplify maintenance this is using now just in memory signer as the `react-native` example.

File: examples/web-wagmi/package.json [hunk 2]
```diff
@@ -15,7 +16,7 @@
     "tsc": "tsc --noEmit"
   },
   "dependencies": {
-    "@bundlr-network/client": "^0.10.3",
+    "@bundlr-network/client": "^0.11.9",
```
Comment (desfero) [inline examples/web-wagmi/package.json:19]: Had to upgrade bundler due to some heresy in build pipline.
https://github.com/vitejs/vite/issues/9703
Older `@bundlr-network/client` was using `near-api-js` which is probably wrongly published and was causing build failures for production `web-wagmi` deployment.

File: examples/web-wagmi/vite.config.ts [hunk 3]
```diff
@@ -11,19 +11,4 @@ export default defineConfig({
     nodePolyfills({ protocolImports: true }),
     visualizer({ open: true }) as PluginOption,
   ],
-
-  build: {
```
Comment (desfero) [inline examples/web-wagmi/vite.config.ts:15]: simplifying, for example app it's not needed

File: examples/nextjs/components/auth/LoginButton.tsx (hunk 1 above)
Comment (reecejohnson) [reply examples/nextjs/components/auth/LoginButton.tsx:5]: nice, thanks!

File: examples/nextjs/domain/wallet.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// This is NOT PRODUCTION READY implementation of the EOA.
```
Comment (reecejohnson) [inline examples/nextjs/domain/wallet.ts:1]: good to show usage of the SDK without `wagmi` 👍

File: examples/web-wagmi/package.json [hunk 5]
```diff
@@ -3,8 +3,9 @@
   "private": true,
   "version": "0.1.0",
   "scripts": {
+    "build:with-packages": "turbo run build --filter=...example-wagmi",
```
Comment (krzysu) [inline examples/web-wagmi/package.json:6]: what does `--filter=...example-wagmi` do?

**Review by krzysu — APPROVED** [review]
nice!

**Review by reecejohnson — APPROVED** [review]

File: examples/web-wagmi/package.json (hunk 5 above)
Comment (desfero) [reply examples/web-wagmi/package.json:6]: Builds all dependencies of the `example-wagmi` inside monorepo, we could also build the whole monorepo but this one is better as only builds what `example-wagmi` is actually using.

File: examples/web-wagmi/vite.config.ts [hunk 6]
```diff
@@ -11,19 +11,4 @@ export default defineConfig({
...
-
-    rollupOptions: {
-      treeshake: {
-        preset: 'recommended',
-      },
-    },
-  },
-
-  define: {
-    'process.env': {},
```
Comment (cesarenaldi) [inline examples/web-wagmi/vite.config.ts:27]: @desfero did we check metadata upload via WebBundlr works still? This line and the use of `nodePolyfills` plugin were needed to make `WebBundlr` works on browsers.

**Pull request merged**